menyer/ラーメン/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;1,300&display=swap">
+    <link rel="stylesheet" href="style.css">
+    <title>ラーメン</title>
+</head>
+
+<body>
+    <header>
+        
+            <img src="img/img_header_1.png" alt="bowle of ramen" width="400px">
+            <span>
+                <h1>ラーメン</h1>
+            </span>
+        
+    </header>
+    <main>
+        <article>
+            <div class="spacing_btw_elements">
+                <h2>SOUPS</h2>
+            </div>
+            <div class="box_1">
+                <h4>MISO RAMEN
+                    <br>
+                    味噌ラーメン
+                </h4>
+                <p>Miso, pork and chicken broth. Chashu pork belly, spicy bean sprouts, boild ramen egg, nori seaweed
+                    and Negi japanese long green onion.</p>
+                <p>120 kr</p>
+            </div>
+            <div class="spacing_btw_elements"></div>
+            <div class="box_1">
+                <h4>TSUKEMEN DIPPING RAMEN
+                    <br>
+                    つけ麺
+                </h4>
+                <p>Fresh noodles, sliced pork belly, ginger, garlic, shiitake and shimeji mashrooms. Served width a
+                    separate bowl of flavorful broth for dipping </p>
+                <p>135 kr</p>
+            </div>
+            <div class="spacing_btw_elements"></div>
+            <div class="box_1">
+                <h4>VEGETARIAN RAMEN
+                    <br>
+                    豆乳味噌ラーメン
+                </h4>
+                <p>Dashi broth. Fresh noodles, homemade kimchi, shiitake mashrooms, bean sprouts, Doubanjiang chili bean
+                    sauce, toasted sesame seeds, sesame oil</p>
+                <p>110 kr</p>
+            </div>
+            <div class="spacing_btw_elements">
```

Comment:
Your h1,h2,h3.. also should have margin instead of being wrapped with a spacing div.